CMakeLists.txt

```diff
@@ -374,8 +372,20 @@ if(ACTS_BUILD_PLUGIN_JSON)
     endif()
 endif()
 if(ACTS_BUILD_PLUGIN_GEOMODEL)
-    find_package(GeoModelCore ${_acts_geomodel_version} REQUIRED CONFIG)
-    find_package(GeoModelIO ${_acts_geomodel_version} REQUIRED CONFIG)
+    foreach(_gm_ver 7.0.0;6.3.0)
+        message(STATUS "Trying to find GeoModel version ${_gm_ver}")
+        find_package(GeoModelCore ${_gm_ver} CONFIG)
+        find_package(GeoModelIO ${_gm_ver} CONFIG)
```

Comment:
So, did the version range formalism not work then? I was hoping that this could've been written just as something like:

```cmake
find_package(GeoModelCore 6.3...7.99)
```

I'd really like to avoid such custom for-loops if at all possible... 🤔

Reply:
Yeah so the range formalism (for some reason) only allows ranges within a single MAJOR version it seems.

Reply:
That's very disappointing. If we can't write a possibly quite wide range, then what's the point of this feature even? 😕

I do wonder that once we have an entire for-loop in our code to make all this happen, is the version check actually worth it? Then again, I was never really fond of such version checks to begin with... 🤔

Reply:
I have to admit I also hack the versions from time to time. Alternatively we could search without version constrain and then check the version manually afterwards

Reply:
My understanding is that if you supply a range, both ends of the range have to be deemed *compatible* by the target version, i.e. you can't span a range over MAJOR versions, which is indeed disappointing.

I really do want to have a version check, but indeed it might be better to manually check the version after finding without a constraint. I think this only makes a difference if multiple versions are found and CMake would otherwise pick a compatible version out of the available ones.